components/terrain/quadtreeworld.cpp

```diff
@@ -285,7 +295,31 @@ void traverseToCell(QuadTreeNode* node, ViewData* vd, int cellX, int cellY)
     }
 }
 
-unsigned int getLodFlags(QuadTreeNode* node, int ourLod, ViewData* vd)
+/// get the level of vertex detail to render this node at, expressed relative to the native resolution of the data set.
+unsigned int getVertexLod(QuadTreeNode* node, int vertexLodMod)
+{
+    int lod = Log2(int(node->getSize()));
+    if (vertexLodMod > 0)
+    {
+        lod = std::max(0, lod-vertexLodMod);
+    }
+    else if (vertexLodMod < 0)
+    {
+        float size = node->getSize();
+        // stop simplyfing at this level since smaller chunks are already less dense compared to the farther stuff in view relative terms
+        // the cutoff is 1 because at size = 1 a node has getCellVertices() (ie 65) vertices just like larger nodes 
```

Comment:
Explanation (in comments) could be clarified altogether, but at least simplyfing -> simplifying. Thinking of better way to say "less dense compared to the farther stuff in view", but I wonder if I understand the code correctly... My personal preference wouble be to have the comment before the first "if" that is related to the comment (line 306 in this case), but not sure if that's actually better.